packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -2998,29 +2998,22 @@ export interface _Identifiers {
 }
 
 // @beta
-export const ImageGallery: (props: ImageGalleryProps) => JSX.Element;
+export const ImageOverlay: (props: ImageOverlayProps) => JSX.Element;
 
 // @beta
-export interface ImageGalleryImageProps {
+export interface ImageOverlayProps {
     altText?: string;
-    downloadFilename: string;
-    imageUrl: string;
-    title?: string;
-    titleIcon?: JSX.Element;
-}
-
-// @beta
-export interface ImageGalleryProps {
-    images: Array<ImageGalleryImageProps>;
+    imageSrc: string;
     isOpen: boolean;
     onDismiss: () => void;
+    onDownloadButtonClicked: (imageSrc: string) => void;
```

Comment:
Can introduce this in the future, but provide the ability to configure whether or not users will even see the download button